docs/branding/patterns-illustrations.md

```diff
@@ -0,0 +1,181 @@
+---
```

Comment:
Delete this file as there is no meaningful content.

Reply:
I'm working on the files without the content, I will complete all of them and push a commit to this PR once they are done 💯